packages/pages/src/components/link/types.ts

```diff
@@ -9,16 +13,77 @@ export const LinkType = {
 
 /**
  * Type of link types that might be received from the platform.
+ *
+ * @public
  */
 export type LinkType = typeof LinkType[keyof typeof LinkType];
 
 /**
  * Type for CTA field
  * Note that when coming from the platform the label will always be a string
  * but ReactNode allows for more general usage.
+ *
+ * @public
  */
 export interface CTA {
   link: string;
   label?: React.ReactNode;
   linkType?: LinkType;
 }
+
+/**
+ * Configuration options available for any usages of the Link component.
+ */
+interface LinkConfig
+  extends React.DetailedHTMLProps<
+    React.AnchorHTMLAttributes<HTMLAnchorElement>,
+    HTMLAnchorElement
+  > {
+  obfuscate?: boolean;
+  eventName?: string;
+  conversionDetails?: ConversionDetails | undefined;
+}
+
+/**
+ * The shape of the data passed to {@link Link} when directly passing an HREF to the Link component.
+ *
+ * @public
```

Comment:
do we generate docs from these, e.g. via API extractor?

Reply:
I believe it's in preparation. The extractor is set-up

Reply:
do we need to run it then to extract the docs from these new imports?

Reply:
no we don't have to do anything